### PR TITLE
Update 2

### PR DESCRIPTION
**Pull request by JunoNgx**
## Overview

Some minor tuning and readjustment I missed out. Hopefully the last one before I feel comfortable leaving it as it is.

## Changelog

* Add width constraint for the sessions/pricing section for less jarring visuals, as the cards are currently too wide. This only affects the desktop/large screen layout.
* Enlarge the collapsible links from `navbar` to make it more comfortable to select on a smartphone touchscreen.
* Remove the top padding from the banner/navbar area.
* Add margin to `big-heading` ("Data Structures and Abstractions") for better legibility.

As usual, let me know if I missed anything.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -142,6 +142,11 @@ p {
   padding: 100px;
 }
 
+.white-section .sessions {
+  margin: auto;
+  max-width: 1024px;
```
Comment (ict283) [inline css/styles.css:147]: Width limit good

File: css/styles.css [hunk 2]
```diff
@@ -262,6 +262,10 @@ footer>.container-fluid {
     padding: 0;
   }
 
+  .nav-link {
+    font-size: 1.5em;
```
Comment (ict283) [inline css/styles.css:266]: Much needed change here

**Review by ict283 — CHANGES_REQUESTED** [review]
Small change here. Set <p> text to uniform grey colour. Rest looks good! Thanks
Screenshot 2021-02-11 at 7 14 20 PM

**Comment by JunoNgx** [comment]
> 
> 
> Small change here. Set paragraph text to uniform grey colour. This quote is meant to be subtle. Rest looks good! Thanks
> Screenshot 2021-02-11 at 7 14 20 PM

Change pushed with improvement in the semantic use of html tags. Though the visual appearance has generally been reverted, the use of the tags were corrected.

**Pull request merged**